src/test/java/com/auth0/jwk/JwkTest.java

```diff
@@ -44,7 +46,16 @@ public void shouldBuildWithMap() throws Exception {
     @Test
     public void shouldReturnPublicKey() throws Exception {
         final String kid = randomKeyId();
-        Map<String, Object> values = publicKeyValues(kid);
+        Map<String, Object> values = publicKeyValues(kid, KEY_OPS_ARRAY);
+        Jwk jwk = Jwk.fromValues(values);
+
+        assertThat(jwk.getPublicKey(), notNullValue());
+    }
+
+    @Test
+    public void shouldReturnPublicKeyForStringKeyOps() throws Exception {
+        final String kid = randomKeyId();
+        Map<String, Object> values = publicKeyValues(kid, KEY_OPS_STRING);
         Jwk jwk = Jwk.fromValues(values);
 
         assertThat(jwk.getPublicKey(), notNullValue());
```

Comment:
String ops will be converted to a list, so add `assertThat(jwk.getOperations(), is(KEY_OPS_ARRAY))` or `listContaining..(KEY_OPS_ARRAY)`.